pkgs/by-name/ga/galaxy-dummy-service_0_2_0/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  comet-gog_0_2_0,
+  meson,
+  ninja,
+  pkgsCross,
+}:
+let
+  comet = comet-gog_0_2_0;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "galaxy-dummy-service";
+  inherit (comet) version src;
+  sourceRoot = "${finalAttrs.src.name}/dummy-service";
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgsCross.mingwW64.buildPackages.gcc
```

Comment:
Is this using `pkgsCross` (& friends) allowed in nixpkgs? I don't remember the policy...

Reply:
You might be able to find an answer if you ask in the Nix Cross matrix channel

Reply:
Alright